app/oauth.go

```diff
@@ -501,6 +501,10 @@ func (a *App) DeauthorizeOAuthAppForUser(userID, appID string) *model.AppError {
 		}
 	}
 
+	if err := a.Srv().Store().OAuth().RemoveAuthDataByClientId(appID, userID); err != nil {
+		return model.NewAppError("DeauthorizeOAuthAppForUser", "app.oauth.RemoveAuthDataByClientId.app_error", nil, "", http.StatusInternalServerError).Wrap(err)
```

Comment:
It looks like this translation is missing. Other than that, LGTM.

Reply:
+1 the source translation string (in `en.json`) is missing